tests/e2e/tests/checkout/checkout-block.merchant.block_theme.spec.ts

```diff
@@ -17,100 +17,402 @@ const blockData: BlockData = {
 	},
 };
 
-test.describe( 'Merchant → Checkout', () => {
+test.describe( 'Merchant → Checkout block → In page editor', () => {
 	// `as string` is safe here because we know the variable is a string, it is defined above.
 	const blockSelectorInEditor = blockData.selectors.editor.block as string;
 
-	test.describe( 'in page editor', () => {
-		test.beforeEach( async ( { editorUtils, admin, editor } ) => {
-			await admin.visitSiteEditor( {
-				postId: 'woocommerce/woocommerce//page-checkout',
-				postType: 'wp_template',
-			} );
-			await editorUtils.enterEditMode();
+	test.beforeEach( async ( { editorUtils, admin, page, editor } ) => {
+		await admin.visitSiteEditor( {
+			postId: 'woocommerce/woocommerce//page-checkout',
+			postType: 'wp_template',
+		} );
+		await editorUtils.enterEditMode();
+		await editor.openDocumentSettingsSidebar();
+
+		await page
+			// Spinner was used instead of the progress bar in an earlier version of
+			// the site editor.
+			.locator( '.edit-site-canvas-loader, .edit-site-canvas-spinner' )
+			// Bigger timeout is needed for larger entities, for example the large
+			// post html fixture that we load for performance tests, which often
+			// doesn't make it under the default 10 seconds.
+			.waitFor( { state: 'hidden', timeout: 60_000 } );
```

Comment:
This PR https://github.com/woocommerce/woocommerce-blocks/pull/11866, fixed the spinner bug. We can remove the following code:

```JS
await page
	// Spinner was used instead of the progress bar in an earlier version of
	// the site editor.
	.locator( '.edit-site-canvas-loader, .edit-site-canvas-spinner' )
	// Bigger timeout is needed for larger entities, for example the large
	// post html fixture that we load for performance tests, which often
	// doesn't make it under the default 10 seconds.
	.waitFor( { state: 'hidden', timeout: 60_000 } );
```

Reply:
Nice, thanks!